validator/docs/schemas.md

```diff
@@ -0,0 +1,163 @@
+
+```yaml
+app: my-application   # Application's name
+is_enabled: false     # Is application deployment enabled
+git: git@microservice_repo # Git repository URL
+variables:            # Environment variables
+  app:
+    APP_VAR1: application_variable_1
+  ope:
+    OPE_VAR1: ope_variable_1
+```
+
+To customize your application deployment, there are other sections supported. Check the following sections.
+
+### Processes
+
+Define processes of a project. A project can contain multiple processes such as workers, web processes and more.
+The structure to define a process is the following:
+
+```yaml
+processes:
+  - name: process-name  # Process name, STRING - Required
+    is_cronjob: false   # Defining a simple process, BOOLEAN - Required
+    start_command: npm start  # Executed command when the pod is started. STRING - Required
+  - name: cronjob-name  # Process name - Required
+    is_cronjob: true    # Defining a scheduled process (cronjob) - Required
+    start_command: npm run worker # Executed command when the pod is started - Required
+```
+
+### Scales
+When you deploy an application in a Kubernetes environment, you define how many replicas you would like to run. 
+Each replica represents a Kubernetes POD that encapsulates your application container. 
+And also you can increase, or reduce the number of replicas in the given limit.
+
+```yaml
+scales:
+  my-process:         # Process to scale
+    min-replicas: 1   # Minimal number of pods, INTEGER - Required
+    max-replicas: 10  # Minimal number of pos, INTEGER - Required
+    targetCPUUtilizationPercentage: # Percentage of CPU which will trigger a horizontal scaling when it is exceeded, INTEGER, 0 <= x <= 100 - Required
+  another-process: ...
+```
+
+### Crons
+Crons are tasks that run at a specific time or interval. You can use this type of features to automate tasks, commonly as: reporting, backups, maintenance, emails and so on.
+
+You can define the crons information with the following structure.
+
+```yaml
+crons:
+  my-cronjob: # Cronjob to define, it must be the same name as the cronjob defined in processes
+    concurrency_policy: 'Forbid' # Allow, Forbid or Replace concurrency of cronjobs, STRING - Required. [Kubernetes documentation](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#concurrency-policy)
+    schedule: '* * * * *' # STRING - Required
+    suspend:              # BOOLEAN
+  another-resource: ...
+```
+
+### Resources
+This structure defines the resources to be allocated for your application.
+
+```yaml
+resources:
+  my-resource:  # Name of the resource to define
+    limit:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    limits:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    request:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    requests:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+  another-resource: ...
+```
+
+
+### TemplateVars
```

Comment:
I am not sure what these variables are used for, I am wondering if we should not omit them first, to keep it simple and see afterwards if we need them?

Reply:
These variables from my understanding are used to override default values for Kubernetes. I will add the explanation in the introductory paragraph so it's clearer 👍